modules/BuildUtilities/src/server/PrepareConfigFiles.coffee

```diff
@@ -185,16 +185,16 @@ getProperties = (configDir) =>
 		allConf = _.extend allConf, propertiesParser.read(configFile)
 		console.info "read conf file: #{configFile}"
 
-	configString = ""
+	confObject = {}
```

Comment:
I was concerned that a basic dict / object will not be guaranteed to retain order:
https://stackoverflow.com/questions/5525795/does-javascript-guarantee-object-property-order
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

It seems like a `Map` object might be a better choice for storing the configs and ensuring the order matches how they were read in.

Reply:
Thanks, I switched this over to use `Map`